tests/results/a3f571300587d7e8274699f27677b2883522e8d42677c2788c9dcd696a677420_perdu_extra_space/kangooroo/results.json

```diff
@@ -29,13 +27,13 @@
     },
     "requestedUrl": {
       "url": "http://perdu.com",
-      "urlMd5": "75d53866bcb20465b3287cf237234464",
+      "urlMd5": "8a9c26db8a9d5ad647bd642887c4bfd2",
```

Comment:
This is not the md5 of the value contained in the url key. I think this can be very puzzling.